options/locale/locale_zh-CN.ini

```diff
@@ -30,6 +30,7 @@ captcha=验证码
 twofa=两步验证
 twofa_scratch=两步验证口令
 passcode=验证码
+use_passcode=使用验证码
```

Comment:
Gitea uses Crowdin for translations. If this PR is accepted then the option to add the translations will be added to be translatable there.

See https://github.com/go-gitea/gitea/blob/master/CONTRIBUTING.md#translation for more information.